packages/core/src/tools/instrumentMethod.spec.ts

```diff
@@ -54,6 +54,27 @@ describe('instrumentMethod', () => {
     expect(instrumentationSpy).toHaveBeenCalled()
   })
 
+  it('does not throw if the object is frozen', () => {
+    const object = { method: () => 1 }
+    Object.freeze(object)
+
+    expect(() => {
+      instrumentMethod(object, 'method', () => () => 2)
+    }).not.toThrow()
+  })
+
+  it('does not throw if the method is not writable', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should we document the case where the method is readonly but inherited from the prototype?

Reply:
I added some documentation to the `instrumentMethod` function.